test/no-multiple-inheritance.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @fileoverview Tests for no-multiple-inheritance rule
+ * @author Nicolas Feignon <nfeignon@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require ('solium');
+
+var userConfig = {
+    rules: {
+        "security/no-multiple-inheritance": "error"
+    }
+};
+
+describe ('[RULE] no-multiple-inheritance: Acceptances', function () {
+
+    it ('should accept programs that don\'t use multiple inheritance', function (done) {
```

Comment:
Add another test --> where a contract inherits from a single contract. Since this rule checks for >1, =1 is an acceptable condition.